playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -238,3 +241,26 @@ def get_resource_yaml(event: KubernetesResourceEvent):
     except Exception as exc:
         logging.error("Unexpected error occurred!")
         logging.exception(exc)
+
+
+class NamedResourcesParams(ActionParams):
+    """
+    :var kind: The k8s resource kind. Must be one of: [node,deployment,statefulset,daemonset,job,persistentvolume,persistentvolumeclaim,service,configmap,networkpolicy].
+    :var namespace: For namespaced k8s resources. List names for the specified namespace. If omitted, all namespaces will be used
+    """
+
+    kind: str
+    namespace: Optional[str]
+
+
+@action
+def list_resource_names(event: ExecutionBaseEvent, params: NamedResourcesParams):
```

Comment:
@arikalon1 should we add this to light actions? if it just lists the names it should be okay.

Reply:
yes! Good catch
It's a must